<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-5DGV9QA" name="Regression suite 1" tests="25" failures="3" timestamp="8 Mar 2024 01:49:58 GMT" time="0.268" errors="0">
  <testcase name="funA" time="0.015" classname="TestngFramewokDemo.TestngDemo1"/>
  <testcase name="funD" time="0.001" classname="TestngFramewokDemo.TestngDemo2"/>
  <testcase name="funA" time="0.001" classname="TestngFramewokDemo.TestngDemo2"/>
  <testcase name="funB" time="0.001" classname="TestngFramewokDemo.TestngDemo2"/>
  <testcase name="funC" time="0.002" classname="TestngFramewokDemo.TestngDemo2"/>
  <testcase name="funB" time="0.002" classname="TestngFramewokDemo.TestngDemo3"/>
  <testcase name="funD" time="0.005" classname="TestngFramewokDemo.TestngDemo3"/>
  <testcase name="funA" time="0.001" classname="TestngFramewokDemo.TestngDemo3"/>
  <testcase name="funC" time="0.001" classname="TestngFramewokDemo.TestngDemo3"/>
  <testcase name="CustomerDetails" time="0.001" classname="TestngFramewokDemo.TestngDemo4"/>
  <testcase name="loginTest" time="0.001" classname="TestngFramewokDemo.TestngDemo4"/>
  <testcase name="loginTest" time="0.001" classname="TestngFramewokDemo.TestngDemo4A"/>
  <testcase name="CustomerDetails" time="0.001" classname="TestngFramewokDemo.TestngDemo4A"/>
  <testcase name="loginTest" time="0.001" classname="TestngFramewokDemo.TestngDemo4B">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at TestngFramewokDemo.TestngDemo4B.loginTest(TestngDemo4B.java:11)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="CustomerDetails" time="0.001" classname="TestngFramewokDemo.TestngDemo4B"/>
  <testcase name="loginTest" time="0.001" classname="TestngFramewokDemo.TestngDemo4C">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at TestngFramewokDemo.TestngDemo4C.loginTest(TestngDemo4C.java:11)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="CustomerDetails" time="0.0" classname="TestngFramewokDemo.TestngDemo4C">
    <skipped/>
  </testcase> <!-- CustomerDetails -->
  <testcase name="funA" time="0.001" classname="TestngFramewokDemo.TestngDemo5"/>
  <testcase name="funB" time="0.001" classname="TestngFramewokDemo.TestngDemo5">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at TestngFramewokDemo.TestngDemo5.funB(TestngDemo5.java:16)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- funB -->
  <testcase name="funC" time="0.001" classname="TestngFramewokDemo.TestngDemo5">
    <skipped/>
  </testcase> <!-- funC -->
  <testcase name="funA" time="0.0" classname="TestngFramewokDemo.TestngGroupDemo1"/>
  <testcase name="funB" time="0.002" classname="TestngFramewokDemo.TestngGroupDemo1"/>
  <testcase name="funC" time="0.0" classname="TestngFramewokDemo.TestngGroupDemo1"/>
  <testcase name="funD" time="0.001" classname="TestngFramewokDemo.TestngGroupDemo1"/>
  <testcase name="f" time="0.001" classname="TestngFramewokDemo.TestngXmlDemo1"/>
</testsuite> <!-- Regression suite 1 -->
